src/1_has_value_checker.js

```diff
@@ -1,3 +1,11 @@
-module.exports = function hasValue() {
-  // TODO Implement me.
-};
+/* module.exports = */
+
+function hasValue (obj, value) {
+
+  if (Object.values(obj).includes(value)) {
+   return true;
+}
+  else {
+    return false;
+}
```

Comment:
może być uproszczone:
```suggestion
  return (Object.values(obj).includes(value))
```